backend/question-service/src/middleware/auth.ts

```diff
@@ -21,8 +37,7 @@ export const authMiddleware = async (
   const authRes = await fetch(authEndpoint, {
     method: "POST",
     headers: {
-      ...(req.headers as HeadersInit), // Pass headers from the incoming request
```

Comment:
To give context, this is the line that causes the error. So we need to use another way to pass the header :P